backend/benefit/applications/management/commands/seed.py

```diff
@@ -31,23 +33,32 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         total_created = len(ApplicationStatus.values) * options["number"]
-        if os.getenv("DEBUG") != "1":
-            self.stdout.write("Seeding is only allowed in debug mode")
+        if not settings.DEBUG:
+            self.stdout.write(
+                "Seeding is allowed only when the DEBUG variable set to True"
+            )
             return
         run_seed(options["number"])
         self.stdout.write(f"Created {total_created} applications")
 
 
 def clear_applications():
-    """Clear all seeded applications and application basis records,
+    """Clear all seeded applications and application basis records and terms,
     which are not deleted with on_delete=CASCADE and delete the media folder"""
-    Application.objects.all().delete()
+    applications = Application.objects.all()
+    for application in applications:
+        for attachment in application.attachments.all():
+            attachment.attachment_file.delete()
+    applications.delete()
+
     ApplicationBasis.objects.all().delete()
-    shutil.rmtree(settings.MEDIA_ROOT)
```

Comment:
I assume this fixes a bug wherer `python manage.py seed --number=0` is used to clear all applications. Good stuff.